axelar-chains-config/info/stagenet.json

```diff
@@ -1492,221 +1497,6 @@
       "finality": "finalized",
       "approxFinalityWaitTime": 25
     },
-    "test-avalanche": {
```

Comment:
Are we killing `test-avalanche` and `test-sepolia`? Should we disable them in our config, too?

If possible, can we keep at least one of them so that we have at least two Amplifier chains in stagenet for testing Amplifier -> Amplifier calls?